site/docs-md/guides/deep-links.md

```diff
@@ -0,0 +1,311 @@
+
+```typescript
+const AppUrlListener: React.FC<any> = () => {
+    let history = useHistory();
+    useEffect(() => {
+        CapApp.addListener('appUrlOpen', (data: any) => {
+            // Example url: https://beerswift.app/tabs/tab2
+            // slug = /tabs/tab2
+            const slug = data.url.split(".app").pop();
+            if (slug) {
+                history.push(slug);
+            }
+            // If no match, do nothing - let regular routing 
+            // logic take over
+        });
+    }, []);
+
+    return null;
+};
+
+export default AppUrlListener;
+```
+
+Over in `App.tsx`, import the new component:
+
+```typescript
+import AppUrlListener from './pages/AppUrlListener';
+```
+
+Then add it inside of `IonReactRouter` (or wherever your app is bootstrapped, just ensure that the History hook is available):
+
+```tsx
+const App: React.FC = () => {
+    return (
+        <IonApp>
+            <IonReactRouter>
+                <AppUrlListener></AppUrlListener>
+                <IonRouterOutlet>
+                    <Route path="/home" component={Home} exact={true} />
+                    <Route exact path="/" render={() => <Redirect to="/home" />} />
+                </IonRouterOutlet>
+            </IonReactRouter>
+        </IonApp>
+    );
+};
+```
+
+## Creating Site Association Files
+
+In order for Apple and Google to permit deep links to open your app, a two-way association between your website and app must be created. One file for each must be created and placed within a `.well-known` folder on your website, like so: https://beerswift.app/.well-known/.
+
+Continue on for iOS and Android configuration details.
+
+## iOS Configuration
+
+iOS configuration involves creating a site association file and configuring the native app to recognize the app domain.
+
+> You must be enrolled in the Apple Developer Program.
+
+### Create Site Association File
+
+First, log into the [Apple Developer site](https://developer.apple.com). Navigate to the "Certificates, Identifiers, & Profiles" section and select your app's identifier. Note the Team ID and Bundle ID, and under Capabilities, toggle "Associated Domains" then save:
+
+![iOS Identifier Config](/assets/img/docs/guides/deep-links/ios-config.png)
+
+Next, create the site association file (`apple-app-site-association`).
+
+> Note: Despite being a JSON file, do not save it with a file extension.
+
+```json
+// apple-app-site-association
+{
+    "applinks": {
+        "apps": [],
+        "details": [
+            {
+                // example: 8L65AZE66A.com.netkosoft.beerswift
+                "appID": "TEAMID.BUNDLEID",
+                "paths": ["*"]
+            }
+        ]
+    }
+}
+```
+
+> Note: Some tutorials mention that the file must be served with content type `application/pkcs7-mime`. This is not needed. If your app runs in iOS 9 or later and you use HTTPS to serve the file, you can create a plain text file that uses the `application/json` MIME type.
```

Comment:
I don't think this makes sense, if they are following your tutorial, why talk about other tutorials that are wrong?

Reply:
good call, removed.